deploy/operator/role.yaml

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
There is a lot of stuffhere  that is not needed for the operator like the certificate part, I would say that we just be reactivate here and add after stuff is failing so we have minimal herre, at the end is important to reduce the attack surface of an operator.